src/components/BosLoaderBanner.js

```diff
@@ -40,20 +65,42 @@ export function BosLoaderBanner() {
   if (!redirectMapStore.loaderUrl) return null;
 
   return (
-    <Banner>
-      <div>
-        <p>Loading components from: {redirectMapStore.loaderUrl}</p>
-        {redirectMapStore.failedToLoad && (
-          <p style={{ color: 'red' }}>
-            BOS Loader fetch error, see console logs. CORS errors may be misleading and mean your endpoint cannot be
-            reached
-          </p>
-        )}
-      </div>
-
-      <Button type="button" onClick={closeBanner}>
-        <i className="ph-fill ph-x-circle" />
-      </Button>
-    </Banner>
+    <OverlayTrigger
+      key={"bos-loader"}
+      placement={"bottom"}
+      overlay={
+        <Tooltip>
+          {redirectMapStore.failedToLoad
+            ? "Check console.log. CORS errors may be misleading"
+            : redirectMapStore.loaderUrl}
+        </Tooltip>
+      }
+    >
+      <Floating>
+        <Container href={"/flags"}>
+          {redirectMapStore.failedToLoad
+            ? "BOS Loader fetch error"
+            : "Loading components"}
+        </Container>
+        <Button type="button" onClick={closeBanner}>
+          <i className="bi bi-x" />
+        </Button>
+      </Floating>
+    </OverlayTrigger>
   );
-}
\ No newline at end of file
+}
```

Comment:
Remove commented out code

Reply:
Go ahead and clean out all commented out code in this file (the useEffect, NavigationWrapper, Footer too)